File: src/Aicl.Colmetrik.Host.Web/WebApp/modules/app/model/AuthRole.js
Ext.define('App.model.AuthRole',{
	extend: 'Ext.data.Model',
	idProperty: 'Id',
	fields: 
	[
		{
			name: 'Id',
			type: 'int'
		},
		{
			name: 'Name',
			type: 'string'
		},
		{
			name: 'Directory',
			type: 'string'
		},
		{
			name: 'ShowOrder',
			type: 'string'
		},
		{
			name: 'Title',
			type: 'string'
		}
	]
});

File: src/Aicl.Colmetrik.Model/Types/AuthRole.cs
using System;
using System.ComponentModel.DataAnnotations;
using ServiceStack.Common;
using ServiceStack.DataAnnotations;
using ServiceStack.DesignPatterns.Model;

namespace Aicl.Colmetrik.Model.Types
{
    public partial class AuthRole:IHasId<System.Int32>{

        public AuthRole(){}

        [AutoIncrement]
        public System.Int32 Id { get; set;} 

        [Required]
        [StringLength(30)]
        public System.String Name { get; set;} 

        [StringLength(15)]
        public System.String Directory { get; set;} 
                
        [StringLength(2)]
        public System.String ShowOrder { get; set;} 

        [Required]
        [StringLength(30)]
        public System.String Title { get; set;} 

    }
}